tests/e2e/gcp_e2e.py

```diff
@@ -158,7 +159,7 @@ def test_end_to_end_other_disk(self):
     # Get the analysis VM and attach the evidence disk to forensic
     self.analysis_vm, _ = gcp.StartAnalysisVm(
         project=self.project_id, vm_name=self.analysis_vm_name, zone=self.zone,
-        boot_disk_size=10, cpu_cores=4,
+        boot_disk_size=10, boot_disk_type='pd-standard', cpu_cores=4,
```

Comment:
That's the default value - it would be nice to change this test to use a SSD drive and then add an assertion that confirms that the type of the disk that was just created is `pd-ssd`.